### DISCUSSION TITLE
Filter workflow-instances by correlationId

### DISCUSSION THREAD

**Discussion by ricksearcy** [elsa-workflows/elsa-core#1149] [Q&A]
As a service I would like to be able to query the workflow-instances endpoint by correlationId to get workflow instances that have been related to entities in another system by their Id. 

This does not appear to be supported. 
https://localhost:44391/v1/workflow-instances?correlationId=7cf0839b-03f6-4d12-bfc0-c82f65997c7b 
does not work. 

Is there any support for filtering instances by correlationId?

**Comment by sfmskywalker** [comment]
Not yet, but I think we should add it. Can you create an issue for this? It would be easy to add and great to have.
